pkg/kubelet/pleg/generic.go

```diff
@@ -253,7 +253,7 @@ func (g *GenericPLEG) relist() {
 				needsReinspection[pid] = pod
 
 				continue
-			} else if _, found := g.podsToReinspect[pid]; found {
+			} else {
```

Comment:
Why removing the condition?

Reply:
`delete` is a no-op if the element isn't in the map, so the condition is redundant.